### PR TITLE
Incorporating review comments on Issue #40

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: plugins/arangodbendorsements/arangodb.go [hunk 1]
```diff
@@ -73,9 +73,9 @@ func checkEndPoint(ep string) error {
 func (e *ArangoStore) init(dbparams ArangoDBparams) error {
 	e.dbparams = dbparams
 	if dbparams.ConEndPoint == "" {
-		e.dbparams.ConEndPoint = dEndPoint
+		e.dbparams.ConEndPoint = defaultEP
 	} else if err := checkEndPoint(dbparams.ConEndPoint); err != nil {
-		return fmt.Errorf("init failed no valid connection endpoint: %w", err)
+		return fmt.Errorf("init failed, no valid connection endpoint: %w", err)
```
Comment (thomas-fossati) [inline plugins/arangodbendorsements/arangodb.go:78]: maybe to increase the information content of the error message, you could add the URL that has failed validation

File: plugins/arangodbendorsements/main_test.go [hunk 2]
```diff
@@ -81,7 +81,8 @@ func TestInitStore(t *testing.T) {
 
 			err := fetcher.Init(argList)
 			if err != nil {
-				assert.EqualErrorf(t, err, test.wantErr, "received error got != want (%s, %s)", err.Error(), test.wantErr)
+				//assert.EqualErrorf(t, err, test.wantErr, "received error got != want (%s, %s)", err.Error(), test.wantErr)
```
Comment (thomas-fossati) [inline plugins/arangodbendorsements/main_test.go]: this line should go

Comment (yogeshbdeshpande) [reply plugins/arangodbendorsements/main_test.go]: Yes it was a mistake!, I corrected it immediately!

**Review by thomas-fossati — APPROVED** [review]
thanks for addressing my comments!

**Pull request merged**